.azure-pipelines/azure-pipelines-linux.yml

```diff
@@ -5,7 +5,7 @@
 jobs:
 - job: linux
   pool:
-    vmImage: ubuntu-16.04
```

Comment:
@jeremyleung521 Can you explain this change? Typically we don't want to touch anything that's configured by conda-forge.

Reply:
Can't seem to find the exact error message line at the moment, but it's related to this: https://github.com/actions/virtual-environments/issues/3287 

I got a warning while I was testing earlier and thought i should also bundle the change in.

Reply:
Ok, it actually looks like when I rerendered the feedstock that it wanted to make that change anyway, so I'm fine with it.